projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -38,7 +38,7 @@
           <div
             *ngIf="!truncateText; else truncate"
             class="cx-card-label"
```

Comment:
The change by the customCss is OK, but it is still inline.. I was thinking in something like:
class="cx-card-label color_open"

next to:
.color-open {
  color: #38871F;
}
.color-closed {
  color: #6C7079;
}

Reply:
Do you mean something like this?
https://github.com/SAP/spartacus/pull/16005/commits/6af961183d9297e75ee911b590a972d9b802d87d#diff-c3b9ae880d0032f2fc3b0428a589360e6364ff4d5a195f6bc17bfbaa7d3b364d

Reply:
Because that was my initial implementation but than I changed it because, using the current implementation someone can just pass the color without any changes in the shared component or css.
in the older implementation, we need to add css in the shared components's [css](https://github.com/SAP/spartacus/pull/16005/commits/6af961183d9297e75ee911b590a972d9b802d87d#diff-d05258fec2870a02438f36e3338ce82b7c226fb568aac9812b126d570e31948f) file.

Reply:
OK, we can still use inline colors